servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -812,31 +781,7 @@
             "description": "The report of the node health",
             "required": true,
             "schema": {
-              "type": "object",
-              "description": "The report of the node health",
-              "properties": {
-                "SourceId": {
-                  "type": "string"
-                },
-                "Property": {
-                  "type": "string"
-                },
-                "HealthState": {
-                  "$ref": "#/definitions/HealthState"
-                },
-                "Description": {
-                  "type": "string"
-                },
-                "TimeToLiveInMilliSeconds": {
-                  "type": "string"
-                },
-                "SequenceNumber": {
-                  "type": "string"
-                },
-                "RemoveWhenExpired": {
-                  "type": "boolean"
-                }
-              }
+              "$ref": "#/definitions/NodeHealthReport"
```

Comment:
How much will they change? I assume the changes would be incremental. Incremental changes can be modeled by inheritance `"allof"` at that time to keep the common properties in a base class and use specific model classes that have different properties. 
It will be a breaking change at that time and customers will know about this when you upgrade the major version (semver) of the client library package.